docs/best-practices/auto-scaling.md

```diff
@@ -3,7 +3,7 @@ title: Autoscaling guidance
 titleSuffix: Best practices for cloud applications
 description: Review autoscaling guidance. Autoscaling is the process of dynamically allocating resources to match performance requirements.
 author: EdPrice-MSFT
-ms.date: 05/17/2017
+ms.date: 10/07/2022
```

Comment:
We didn't do a maintenance review this article yet, and should still be in the queue for maintenance.  Please revert the date change.  Thanks.

```suggestion
ms.date: 05/17/2017
```